src/OrleansSQLUtils/OrleansSQLUtils.csproj

```diff
@@ -23,17 +23,17 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="CreateOrleansTables_MySQL.sql">
+    <None Include="CreateOrleansTables_MySql.sql">
```

Comment:
Why changing the casing?

Reply:
I believe it fails on non-Windows builds otherwise.

Reply:
Makes sense, but then we should correct the casing of the files itself, not pascal casing SQL in the csproj IMHO.

Reply:
Yes, because the casing here is incorrect.